
---
string[] room_names
geometry_msgs/Pose[] poses